Cargo.toml

```diff
@@ -14,9 +14,9 @@ edition = "2021"
 
 [dependencies]
 async-channel = "2.2"
-async-trait = "0.1"
 futures-channel = "0.3"
 serde = "1.0"
+trait-variant = { git = "https://github.com/rust-lang/impl-trait-utils.git", rev = "91bd9eb6ab01fee8d64295273e7e40b16a9f60e0" }
```

Comment:
It's on cargo. Not sure why you're pulling from GitHub.

https://crates.io/crates/trait-variant